Sprint-2/errors/0.js

```diff
@@ -1,9 +1,37 @@
 // Predict and explain first...
 
 // call the function capitalise with a string input
-// interpret the error message and figure out why an error is occurring
+// interpret the error message and figure out why an error occurring
 
-function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+
+/*  
+  we are creating a variable with (let = nameVariable) but the same time we can not reassign
+  the value in the same line.
+  instead we should call directly the (str = valueAssigned) and with this will Work
+*/
+
+
+// function capitalize(str) {
+//   let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+//   return str;
+// }
```

Comment:
Thanks for this @cjyuan , yes we have "SyntaxError because the var str has been declared " so to fix this we just assign the value. I update my response. [commit](e2929c210dbc6170a15da1e14b8878052a6fad48)